ament_index_python/setup.py

```diff
@@ -20,9 +20,9 @@
         'Topic :: Software Development',
     ],
     description='Python API to access the ament resource index.',
-    long_description='''\
+    long_description="""\
 A Python API to find resources based on their type in the ament resource index
-and get the content of individual resources.''',
+and get the content of individual resources.""",
```

Comment:
-1 from me, I prefer the `"` when doing multi line strings and doc blocks.

Reply:
Since our current style is not checkable with existing tools I am proposing to use this in order to avoid the need for a manual review.

Reply:
This is actually changing it from triple single quotes to triple double quotes (the same as we use for docblocks). So exactly to what you are preferring :wink:

Reply:
Yup, sorry I misread the diff 😄. I don't even know why I prefer that (or why the plugin would suggest it), but I do like it better.